### ISSUE TITLE
flask_apscheduler.auth on python 3.9

### ISSUE THREAD

**Issue by conal-mediro** [viniciuschiele/flask-apscheduler#165]
I have some code that imports a module from flask_apscheduler.auth
It used to work on an old set of packages for python 3.7
On a fresh install in python 3.9 I get this error:

```
    from flask_apscheduler.auth import HTTPAuth
  File "...\AppData\Local\Programs\Python\Python39\lib\site-packages\flask_apscheduler\auth.py", line 20, in <module>
    from werkzeug.http import bytes_to_wsgi, wsgi_to_bytes
ImportError: cannot import name 'bytes_to_wsgi' from 'werkzeug.http' (...\AppData\Local\Programs\Python\Python39\lib\site-packages\werkzeug\http.py)
```

Is there a way to work around this on newer versions of flask / werkzeug?

**Comment by christopherpickering** [comment]
It looks like they (werkzeug) are put the code inside function where is was used and got rid of those functions. 

```python
def wsgi_to_bytes(data):
    """coerce wsgi unicode represented bytes to real ones"""
    if isinstance(data, bytes):
        return data
    return data.encode("latin1")  # XXX: utf8 fallback?


def bytes_to_wsgi(data):
    assert isinstance(data, bytes), "data must be bytes"
    if isinstance(data, str):
        return data
    else:
        return data.decode("latin1")
```

You can probably remove the import from `flask_apscheduler\auth.py` and add these functions in?

**Comment by viniciuschiele** [comment]
I released a new version 1.12.3, thank you all
